.github/dependabot.yml

```
@@ -95,3 +96,46 @@ updates:
       - "🟩 priority: low"
       - "🧱 stack: mgmt"
       - dependencies
+  - package-ecosystem: "github-actions"
```

Leave a blank line between these two blocks.


```suggestion

  - package-ecosystem: "github-actions"
```